lenses/postfix_virtual.aug

```diff
@@ -32,7 +32,7 @@ let space_or_eol (sep:regexp) (default:string) =
   del (space_or_eol_re? . sep . space_or_eol_re?) default
 
 (* View: word *)
-let word = store /[A-Za-z0-9@\*.+=-]+/
+let word = store /[A-Za-z0-9@\*.+=-_]+/
```

Comment:
You need to keep the dash `-` as the last thing in that character class. The way you have it now, this gets interpreted as 'include all characters from `=` to `_`' which is not what you want.